content/en/docs/_index.md

```diff
@@ -29,5 +29,60 @@ Operators must claim ownership of their APIs. OLM will prevent conflicting Opera
 ### Declarative UI controls
 Operators can behave like managed service providers. Their user interface on the command line are APIs. For graphical consoles OLM annotates those APIs with descriptors that drive the creation of rich interfaces and forms for users to interact with the Operator in a natural, cloud-like way. 
 
+
+## How OLM solves the cluster admin and developer needs.
+
+### Operator Installation and Management
+
+This packaging mechanism helps to simplify the multiple steps involved in deploying an Operator. OLM fulfills necessary metadata for visualizing them incompatible UIs, including installation instructions and API hints in the form of CRD descriptors. It helps cluster admin, developers to manage the lifecycle of all Operators and their associated services running across their clusters.
+
+### Dependency Resolution
+
+Operator developers often require other operators to support their operators. It creates a dependency on the other operators. OLM offers dependency resolution and upgrades lifecycle of running operators. This functionality helps operator developers to resolve the problems related to dependency. As a developer or cluster admin, no need to care about the dependency of my operator.
+
+### Works similar to package Managers
+
+Operator developers may need all APIs for the operator installation. OLM verifies that all required APIs are available for the operator. If they are not available, then OLM will not install the operator. As an operator developer, there is no need to looks for the required APIs. OLM will take care of it.
```

Comment:
Operator developers may need to use one or more APIs for the operator installation and OLM helps solve this use case by verifying that all of the required APIs are available for the operator. If they are not available, then OLM will not install the operator.

Reply:
I modified the #111 PR for the above changes.